pkg/controller/vitessshard/vitessshard_controller.go

```diff
@@ -184,6 +186,10 @@ func (r *ReconcileVitessShard) Reconcile(request reconcile.Request) (reconcile.R
 	tabletResult, err := r.reconcileTablets(ctx, vts)
 	resultBuilder.Merge(tabletResult, err)
 
+	// Perform rolling updates on tablets if needed.
```

Comment:
Copy the NOTE comment below to here as well since it also applies.

Reply:
Done.